scripts/DGSPlanner/DGSPlannerGUI.py

```diff
@@ -82,6 +83,8 @@ def __init__(self, ol=None, parent=None):
         self.trajfig = Subplot(self.figure, 1, 1, 1, grid_helper=self.grid_helper)
         self.trajfig.hold(True)
         self.figure.add_subplot(self.trajfig)
+        self.toolbar = NavigationToolbar(self.canvas, self)
```

Comment:
I would create a vertical layout, add the canvas widget to it, then the navigation toolbar, and add this layout to `self.layout()`.
In addition, you should remove the save button, or override the `save_figure` function of the navigation toolbar to run the save function of DGSPlannerGUI widget. This will add information about instrument/goniometer/UB, integration ranges